core/toolbox.js

```diff
@@ -684,13 +684,24 @@ Blockly.Toolbox.Category.prototype.dispose = function() {
   this.contents_ = null;
 };
 
+Blockly.Toolbox.Category.prototype.getMenuItemClassName = function(selected) {
```

Comment:
Can you add the appropriate JSDoc for this method?

Also, since this returns a list, should the name be plural (i.e. getMenuItemClassNames)?
I think this can also be marked private and therefore should have an _ at the end of the method name.

Reply:
This method actually just returns a string of space-separated classes, not a list.
Can it stay singular then?